astro/src/content/docs/extend/examples/multi-application-dashboard.mdx

```diff
@@ -0,0 +1,166 @@
+
+- Click the <IconButton name="edit" /> edit button in the Bank theme row's action column.
+- Select the <InlineUIElement>Index</InlineUIElement> page (fourth item in the list on the left).
+- Paste the code below into the text box.
+  ```html
+  [#ftl/]
+  [#-- @ftlvariable name="tenant" type="io.fusionauth.domain.Tenant" --]
+  [#-- @ftlvariable name="tenantId" type="java.util.UUID" --]
+  [#-- @ftlvariable name="theme" type="io.fusionauth.domain.Theme" --]
+  [#import "_helpers.ftl" as helpers/]
+
+  [@helpers.html]
+    [@helpers.head title="FusionAuth"/]
+    [@helpers.body]
+      [@helpers.header]
+        [#-- Custom header code goes here --]
+      [/@helpers.header]
+
+       [#if theme.type != "simple"]
+          [@helpers.main title="" rowClass="row center-xs" colClass="col-xs-12 col-sm-12 col-md-10 col-lg-10 col-xl-9"]
+              <div class="p-3 pb-5">
+                <div style="width: 180px;">
+                  <img src="/images/logo-gray.svg">
+                </div>
+                <br />
+                <a href="http://localhost:3000/account" style="background: linear-gradient(to bottom right, palegreen, forestgreen);
+                  color: white;
+                  padding: 10px;
+                  margin: 20px;
+                  border-radius: 5px;
+                  text-align: center;
+                  width: 160px;
+                  display: inline-block;
+                  text-decoration: none;">
+                  Changebank
+                </a>
+                <a href="http://localhost:3001/account" style="background: linear-gradient(to bottom right, skyblue, navy);
+                  color: white;
+                  padding: 10px;
+                  margin: 20px;
+                  border-radius: 5px;
+                  text-align: center;
+                  width: 160px;
+                  display: inline-block;
+                  text-decoration: none;">
+                  Changeinsurance
+                </a>
+              </div>
+          [/@helpers.main]
+        [/#if]
+
+      [@helpers.footer]
+        [#-- Custom footer code goes here --]
+      [/@helpers.footer]
+    [/@helpers.body]
+  [/@helpers.html]
+  ```
+- Click the <IconButton name="save" /> save button at the top right.
+
+Besides a little CSS for styling, the code above has only two changes: Adding hyperlink `<a href="http://localhost:3000/account">Changebank</a>` elements for the two applications. Note that the links don't point to the login page of the target application. Instead, they point to the logged-in main page. If the user isn't logged in to FusionAuth, they will be automatically redirected to the app's login page. If they are already logged in, they will be taken straight to the app. This saves the user time. (To see how this authentication check in the app works, look at `bankApp/routes/index.js` in the sample repository you downloaded.)
```

Comment:
```suggestion
Besides a little CSS for styling, the code above only adds links for the two applications. The links don't point to the login page of the target application. Instead, they point to the logged-in main page. If the user isn't logged in to FusionAuth, they will be automatically redirected to the app's login page. If they are already logged in, they will be taken straight to the app. This saves the user time while still protecting the application content.
```

Reply:
Please also remote code include `bankApp/routes/index.js` and show how it works, rather than asking the user to look at it.